mne/externals/pymatreader/pymatreader.py

```diff
@@ -130,12 +133,15 @@ def _hdf5todict(hdf5_object, variable_names=None, ignore_fields=None):
         return_dict = dict()
 
         for key in all_keys:
-            return_dict[key] = _hdf5todict(hdf5_object[key], variable_names=None, ignore_fields=ignore_fields)
+            return_dict[key] = _hdf5todict(hdf5_object[key],
+                                           variable_names=None,
+                                           ignore_fields=ignore_fields)
 
         return return_dict
     elif isinstance(hdf5_object, h5py.Dataset):
         data = hdf5_object.value
-        if isinstance(data, numpy.ndarray) and data.dtype == numpy.dtype('object'):
+        if isinstance(data, numpy.ndarray) and data.dtype == numpy.dtype(
```

Comment:
why did we change this line?

if needed to change, I like better to break after `and \`